presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -809,7 +809,13 @@ public Void visitTableScan(TableScanNode node, Void context)
         @Override
         public Void visitValues(ValuesNode node, Void context)
         {
-            NodeRepresentation nodeOutput = addNode(node, "Values");
+            NodeRepresentation nodeOutput;
+            if (node.getValuesNodeLabel().isPresent()) {
+                nodeOutput = addNode(node, format("Values converted from TableScan[%s]", node.getValuesNodeLabel().get()));
```

Comment:
Should we just say "Values from TableScan" ?

Reply:
I assume that we can have corner case where one table appears in multiple scans which end up with converting to ValuesNode, Maybe output more information can be helpful?